astroquery/nasa_exoplanet_archive/core.py

```diff
@@ -115,11 +142,24 @@ def query_criteria_async(self, table, get_query_payload=False, cache=None, **cri
         References
         ----------
 
-        .. [1] `NASA Exoplanet Archive API Documentation
+        .. [1] `NASA Exoplanet Archive TAP Documentation
+           <https://exoplanetarchive.ipac.caltech.edu/docs/TAP/usingTAP.html>`_
+        .. [2] `NASA Exoplanet Archive API Documentation
            <https://exoplanetarchive.ipac.caltech.edu/docs/program_interfaces.html>`_
         """
+        # Make sure table is lower-case
         table = table.lower()
 
+        # Warn if old table is requested
+        if table in ["exoplanets", "exomultpars"]:
+            # warnings.warn("The '{0}' table is stale and will be depracated in the Archive 2.0 release. Use the 'ps' table. See https://exoplanetarchive.ipac.caltech.edu/docs/ps-pscp_release_notes.html".format(table), InputWarning, )
+            raise InvalidTableError("The ``{0}`` table is no longer updated and has been replaced by the Planetary Systems table (PS), which is connected to the Exoplanet Archive TAP service. Although the argument keywords of the called method should still work on the new table, the allowed values could have changed since the database column names have changed; this document contains the current definitions and a mapping between the new and deprecated names: https://exoplanetarchive.ipac.caltech.edu/docs/API_PS_columns.html. You might also want to review the TAP User Guide for help on creating a new query for the most current data: https://exoplanetarchive.ipac.caltech.edu/docs/TAP/usingTAP.html.".format(table))
+
+        # Warn if old table is requested
+        if table in ["compositepars"]:
```

Comment:
It might be worth combining this with the above condition and just mapping the updated table names using a dict or something?